src/parser/paladin/retribution/modules/talents/Inquisition.js

```diff
@@ -18,20 +19,29 @@ class Inquisition extends Analyzer {
   constructor(...args) {
     super(...args);
     this.active = this.selectedCombatant.hasTalent(SPELLS.INQUISITION_TALENT.id);
+
+    // event listeners
+    this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.INQUISITION_TALENT), this.onInquisitionCast);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(ABILITIES_AFFECTED_BY_DAMAGE_INCREASES), this.onAffectedDamage);
   }
 
-  on_byPlayer_damage(event) {
-    const spellId = event.ability.guid;
-    if (!ABILITIES_AFFECTED_BY_DAMAGE_INCREASES.includes(spellId)) {
-      return;
+  onInquisitionCast(event) {
+    if (this.selectedCombatant.hasBuff(SPELLS.AVENGING_WRATH.id)) {
+      event.meta = event.meta || {};
+      event.meta.isInefficientCast = true;
+      event.meta.inefficientCastReason = 'You refreshed Inquisition during Avenging Wrath';
```

Comment:
You might want to include some info about why you shouldnt do this, especially for those new to the spec.